sale_order_line_remove/readme/CONTRIBUTORS.rst

```diff
@@ -0,0 +1 @@
+* Joan Sisquella <joan.siksquella@forgeflow.com>
```

Comment:
Is your email correct? Sorry it it's right 😇

Reply:
Fixed! Thank you :smile: